content/applications/websites/ecommerce/checkout_payment_shipping/checkout.rst

```diff
@@ -0,0 +1,144 @@
+------------
+
+Promo code
+~~~~~~~~~~
+
+If you have enabled :guilabel:`Discounts, Loyalty, & Gift Card` in the settings
+(:menuselection:`Configuration --> Settings --> Shop - Products`), you can enable a
+:guilabel:`Promo Code` field (:menuselection:`Edit --> Customize`) from any checkout page.
+Customers can then redeem gift cards and promotional codes at the :guilabel:`Review Order` step.
+
+.. image:: checkout/checkout-promo.png
+   :align: center
+   :alt: Gift card and promotion field
+
+Discount subtotal
+~~~~~~~~~~~~~~~~~
+
+If :guilabel:`Discounts, Loyalty, & Gift Card` is enabled, you can display the **subtotal** after
+discount at the :guilabel:`Review Order` step. From any checkout page, go to :menuselection:`Edit
+--> Customize` and enable :guilabel:`Show Discount in Subtotal`.
+
+.. image:: checkout/checkout-subtotal.png
+   :align: center
+   :alt: Subtotal discount
+
+Address
+-------
+
+B2B fields
+~~~~~~~~~~
+
+Optional :guilabel:`TIN/VAT` and :guilabel:`Company Name` fields can be added to the
+:guilabel:`Billing Address` form for B2B customers. To add the fields, go to :menuselection:`Edit
+--> Customize` from any checkout page, and enable :guilabel:`Show B2B fields`.
+
+Request extra info (extra step)
+-------------------------------
+
+You can request :guilabel:`Extra Info` from the customer by adding an **extra step** between the
+:guilabel:`Address` and :guilabel:`Confirm Order` steps. The :guilabel:`Extra Info` step is an
+online form linked to the quotation or sales order of the customer. The information added in the
+extra step is available on the quotation or sales order of the customer from the back end, in the
+**Sales** app.
+
+When enabled, you can remove, add, and modify fields of the form by clicking on :guilabel:`Edit` in
+the top-right corner, and then clicking on any of the form's fields. At the bottom of the
+:guilabel:`Customize` menu, under the :guilabel:`Field` section, are all the customization options
+for the fields, as well as the :guilabel:`+ Field` button to add additional fields.
+
+.. image:: checkout/checkout-form.png
+   :align: center
+   :alt: Online form customization
+
+Confirm Order
+-------------
+
+Terms and Conditions
+~~~~~~~~~~~~~~~~~~~~
+
+You can ask customers to agree to the :guilabel:`Terms & Conditions` in order to confirm their order
+by enabling :guilabel:`Accept Terms & Conditions` under :menuselection:`Edit --> Customize` on any
+checkout page. The :guilabel:`Terms & Conditions` prompt is found on the :guilabel:`Confirm Order`
+step.
+
+.. image:: checkout/checkout-terms.png
+   :align: center
+   :alt: Terms and conditions
+
+.. todo:: add section on suggested accessories when that PR is published
+.. todo:: add ref to discounts when published
+
+Add content
+===========
+
+You can use **building blocks** to add content at any step of the checkout process, including the
+**order confirmation** page. To do so, from any **checkout page**, go to :menuselection:`Edit -->
```

Comment:
```suggestion
You can use **building blocks** to add content at any step of the checkout process. To do so, from any **checkout page**, go to :menuselection:`Edit -->
```

Reply:
I would not mention "incuding the order confirmation page" as you state just before "at any step.